### ISSUE TITLE
disableOntov by default

### ISSUE THREAD

**Issue by sinisarudan** [Cha-OS/KnAllEdge#247]
especially in enforced behaviours, like Brainstorming, when it depends on Ontov, some uneducated user could by mistake broke whole logic by changing filter or changing **OR/AND switch**.
Onse solution is to make them disabled for changes, and on click we are asked to confirm their activation for user interactions.
Other (better) option would be to hide Ontov completly and have a button "Show Ontov" that can have a warning:
"This is an advanced tool, you can practice it"
but when in BS or in similar situtation, warning should be:
"This is an advanced tool, if you're not sure how to use, please don't use it in while in Brainstorming"

**Comment by mprinc** [comment]
Yes, this we have to resolve on more structural level, i assume the cleanest way would be that we implement Redux(-like) mechanism that allow or forbid actions based on the state we are in, and that state will be stacked on the state stack and resolved back after finishing it (i.e. BS).

If we do not do it on that systematic and (clean) declarative way, we will endup with a lot of uncontrollable and hidden side efects that will be (super) hard to control

**Comment by mprinc** [comment]
I started discussion here: https://docs.google.com/document/d/1t720l1KfnOBGHx52o48YGj4NMPLXieEumCxBFstypSM/edit#
